### PR TITLE
chore: hyper v1

### LINKED ISSUE #650: Upgrade hyper to 1.0
Renovate has created the base PR which doesn't pass the CI checks.

https://github.com/tailcallhq/tailcall/pull/636

**Technical Requirements**
- Hyper version for both server should be updated to 1.0
- No performance degradation should be observed
- No tests should fail or require modification

### PR DESCRIPTION
**Pull request by ssddOnTop**
**Issue Reference(s):**  
Fixes #650
/claim 650

### CODE REVIEW

File: src/cli/javascript/js_response.rs
Comment (tontinton) [inline src/cli/javascript/js_response.rs:1]: why is this empty file here?

Comment (ssddOnTop) [reply src/cli/javascript/js_response.rs:1]: Might be some issue with merging

Comment (ssddOnTop) [reply src/cli/javascript/js_response.rs:1]: But why are you reviewing a draft pr?

Comment (tontinton) [reply src/cli/javascript/js_response.rs:1]: Was just looking around, didn't notice that it's a draft

**Pull request closed without merging**